ci: build/test windows

zoeyfyi opened the pull request.

zoeyfyi committed:
ci: build/test windows

zoeyfyi committed:
ci: split gdk lib steps

zoeyfyi committed:
ci: remove git due to conflicts

zoeyfyi committed:
ci: change shell for rm step

zoeyfyi committed:
ci: change vs version

zoeyfyi committed:
ci: re-enable msys

zoeyfyi committed:
ci: debug where

zoeyfyi committed:
ci: override bash shell path

zoeyfyi committed:
ci: re-add icons

zoeyfyi committed:
ci: remove MSYS2

zoeyfyi committed:
ci: add shell to rm step

zoeyfyi committed:
ci: re-install git from MSYS2

zoeyfyi committed:
ci: move MSYS2 setup

zoeyfyi committed:
ci: add MSYS2 to PATH

zoeyfyi committed:
ci: debug path

zoeyfyi committed:
ci: actually debug path

zoeyfyi committed:
ci: add to path a different way

zoeyfyi committed:
ci: more debug

zoeyfyi committed:
ci: fix yml

zoeyfyi committed:
ci: stop using numworks/setup-msys2

zoeyfyi committed:
ci: re-add gtk dependencys

zoeyfyi committed:
ci: try again

zoeyfyi committed:
ci: update and use python2

zoeyfyi committed:
ci: fix broken pacman upgrade

zoeyfyi committed:
ci: ditto

zoeyfyi committed:
ci: no-confirm

zoeyfyi committed:
ci: ignore exit code on pacman update

zoeyfyi committed:
ci: again

zoeyfyi committed:
ci: more workarounds

zoeyfyi committed:
ci: ask 20

zoeyfyi committed:
ci: just update package database

zoeyfyi committed:
ci: overwrite

zoeyfyi committed:
ci: one day this will work

zoeyfyi committed:
ci: update pacman?

zoeyfyi committed:
ci: install python2

zoeyfyi committed:
ci: install python 2

zoeyfyi committed:
ci: debug find python2/3

zoeyfyi committed:
ci: search whole drive

zoeyfyi committed:
ci: cd

zoeyfyi committed:
ci: change search path

zoeyfyi committed:
ci: set an env variable

zoeyfyi committed:
ci: debug tree

zoeyfyi committed:
ci: widen path

zoeyfyi committed:
ci: more debug

zoeyfyi committed:
ci: use working-directory

zoeyfyi committed:
ci: more debug

zoeyfyi committed:
ci: back to gvsbuild

zoeyfyi committed:
ci: fix yml

mrbenshef commented:
gtk + windows + v8 + rust = nightmares

zoeyfyi committed:
ci: delete git bin

zoeyfyi committed:
ci: add libs and build!

zoeyfyi committed:
ci: remove old stuff

zoeyfyi committed:
ci: clean up and cache

zoeyfyi committed:
fix: open command error on non-linux machines

zoeyfyi committed:
fix: sourceview on windows

zoeyfyi committed:
ci: cache cargo and fix test

mrbenshef commented:
Somehow this is kinda working, all that’s left us to package it up and never touch this cursed pipeline again...

zoeyfyi committed:
ci: add linux and macos

zoeyfyi committed:
refactor: remove println's

zoeyfyi committed:
fix: opening urls and folders on windows

zoeyfyi committed:
ci: fix windows cacheing

zoeyfyi committed:
chore: fmt

zoeyfyi committed:
ci: change move's to copy's

zoeyfyi committed:
ci: change old step name

For some reason github actions decided to execute the old workflow file, not sure what happened there...

zoeyfyi committed:
ci: use glob

zoeyfyi committed:
ci: debug tree

zoeyfyi committed:
ci: fix yml

zoeyfyi committed:
ci: reenable build and test

zoeyfyi committed:
ci: only cache lib files

zoeyfyi committed:
ci: remove debug

zoeyfyi committed:
ci: merge release and try release and add windows

mrbenshef commented:
Blocked until I can figure out whats wrong with cacheing on windows.

zoeyfyi committed:
ci: fix gvsbuild cache

zoeyfyi committed:
ci: fix gvsbuild cache

zoeyfyi committed:
ci: use exact path

zoeyfyi committed:
ci: debug paths

zoeyfyi committed:
ci: remove trailing slash

zoeyfyi committed:
ci: think I have it this time

zoeyfyi committed:
ci: copy cache setup from ci to release

zoeyfyi committed:
ci: disable cache prep on hit
